ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -389,7 +395,17 @@ def list_entities(
         if self._use_raw_data:
             return resp
 
-        entities = [entity(**t) for t in resp["data"]]
+        if self._skip_on_failure:
+            entities = []
+            for elmt in resp["data"]:
+                try:
+                    entities.append(entity(**elmt))
+                except Exception as exc:
+                    logger.error(f"Error creating entity. Failed with exception {exc}")
+                    continue
+        else:
+            entities = [entity(**elmt) for elmt in resp["data"]]
+
```

Comment:
if skip failures we will continue instantiating other entities and skip over the ones that may not conform to the expected object shape. This will allow the DI workflow to continue processing the entity type.

IMPORTANT: this does not solve the underlying issue that is faced [here](https://github.com/open-metadata/OpenMetadata/issues/12679) for example where a tag entity has not been migrated properly to new constraints -- this is out of the DI scope.

```
Error trying to fetch entity -- 1 validation error for Table
tags -> 2 -> tagFQN -> __root__
  ensure this value has at most 128 characters (type=value_error.any_str.max_length; limit_value=128)
```

Reply:
If we need it in other places then makes sense. I had put it here as a "hidden" functionality as I thought it was an edge cases.